framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -1228,7 +1250,9 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
         // check precondition annotations
         checkPreconditions(node, contractsUtils.getPreconditions(invokedMethodElement));
 
-        checkSuperConstructorCall(node);
+        if (TreeUtils.isSuperCall(node)) {
```

Comment:
Should we rename `TreeUtils.isSuperCall` and `TreeUtils.isThisCall` to something like `isSuperConstructorCall`?
I needed to look around to see that they only apply to constructor invocations. Or am I missing something?

Reply:
Yes, they only apply to constructor invocations, so that rename would make sense.

Reply:
Filed #2391 to remember this.